src/main/java/com/openlattice/data/DataApi.java

```diff
@@ -281,4 +282,17 @@ Integer replaceEntityInEntitySetUsingFqns(
             @Path( ENTITY_SET_ID ) UUID entitySetId,
             @Path( ENTITY_KEY_ID ) UUID entityKeyId,
             @Path( PROPERTY_TYPE_ID ) UUID propertyTypeId );
+
+    /**
+     * Loads a linked entity set breakdown with the selected linked entities and properties.
+     *
+     * @param linkedEntitySetId The id of the linked entity set to load.
+     * @param selection The id of the requested entity.
+     * @return A enttity details object, with property type FQNs as keys.
+     */
+    @GET( BASE + "/" + ENTITY_SET + "/" + SET_ID_PATH + "/" + DETAILED  )
+    Map<UUID, Map<UUID, Map<UUID, Map<FullQualifiedName, Set<Object>>>>> loadLinkedEntitySetBreakdown(
```

Comment:
Did you consider creating a class like `LinkedEntitySetBreakdown` to represent 
`Map<UUID, Map<UUID, Map<UUID, Map<FullQualifiedName, Set<Object>>>>>>`?

Reply:
What would be the advantage of that?

Reply:
Yeah, it doesn't make sense for this object. Will you please document what the keys are for each level?